simulation/simtypes/account.go

```diff
@@ -39,6 +39,17 @@ func (sim *SimCtx) RandomSimAccountWithConstraint(f SimAccountConstraint) (simul
 	return sim.randomSimAccount(filteredAddrs), true
 }
 
+func (sim *SimCtx) RandomSimAccountWithMinCoins(ctx sdk.Context, denom string, amount int64) (simulation.Account, error) {
```

Comment:
Lets make this take in sdk.Coins as arg

Reply:
Addressed here https://github.com/osmosis-labs/osmosis/pull/2182